parking_permits/utils.py

```diff
@@ -178,17 +178,23 @@ def get_permit_prices(
     return permit_prices
 
 
-def get_meta_value(meta_pair_list, meta_pair_key):
+def get_meta_item(meta_pair_list, meta_pair_key):
     return next(
         (
-            meta_pair.get("value")
+            meta_pair
             for meta_pair in meta_pair_list
             if meta_pair.get("key") == meta_pair_key
         ),
         None,
     )
 
 
+def get_meta_value(meta_pair_list, meta_pair_key):
+    item = get_meta_item(meta_pair_list, meta_pair_key)
+    if item:
+        return item.get("value")
```

Comment:
Not sure what "house style" is here, I've often seen `return None` as compulsory to make it clear what happens if nothing else is returned (even though it's technically redundant).

Reply:
Changed this to format of: `return item.get("value") if item else None`